app/assets/javascripts/discourse/app/widgets/post.js

```diff
@@ -928,4 +933,45 @@ export default createWidget("post", {
       kvs.set({ key: "lastWarnedLikes", value: Date.now() });
     }
   },
+
+  didRenderWidget() {
+    if (!this.currentUser || !this.siteSettings.enable_user_tips) {
+      return;
+    }
+
+    this.currentUser.showUserTip({
+      id: "post_menu",
+
+      titleText: I18n.t("user_tips.post_menu.title"),
+      contentText: I18n.t("user_tips.post_menu.content"),
+
+      reference: document.querySelector(
+        ".post-controls .actions .show-more-actions"
+      ),
+      appendTo: document.querySelector(".post-controls"),
```

Comment:
This will select the first `document.querySelector(".post-controls")` visible, right? Do we want to narrow it down to a specific post?